geoserver/lib/2.15.0/gt-s3-geotiff-21.0.jar.readme

```diff
@@ -0,0 +1,8 @@
+This is gt-s3-getotiff-21-SNAPHOT meant to
+replace gt-s3-geotiff-21.0.jar in 
+geoserver/WEB-INF/lib, with fixes
+that didn't make it to the official 21.0 
+GeoTools release from Tue Feb 2019, whilst
+the patch is from Mon Mar 2019 at commit
+c73b886b1, and hence will be available in
+the next 21.1 GeoTools release.
```

Comment:
I'm not very fond of adding binaries to git, but It's fine. Do you confirm that we could remove them later on ? (once GS will have upgraded the GT version, and we will stick with the new one) ?

Reply:
yeah, we just need to upgrade to a version that comes with the patched s3-geotiff.jar. Which, as of today, should be 15.1, I'll give it a try without the extra binaries and update the PR